docs/first-steps/tracking/index.md

```diff
@@ -15,14 +15,6 @@ import { sampleTrackingCode } from '@site/src/components/FirstSteps/sampleTracki
 
 Once your pipeline is set up, you will want to send some events to it. Here’s an overview of the different options.
 
-:::note Limits
-
-Keep in mind that some of our offerings have limits on the number of events you can send:
-* For Try Snowplow, there is cap of 50 events per second. Any events above this cap will be dropped
-* For Community Edition, the default setup is sized for around 100 events per second
```

Comment:
Should we keep limits for Community Edition?

Reply:
They are still listed in the CE pages, so I would say no